src/sklearn_raster/features.py

```diff
@@ -84,16 +84,82 @@ def apply_ufunc_across_features(
         allow_cast: bool = False,
         check_output_for_nodata: bool = True,
         keep_attrs: bool = False,
+        inner_thread_limit: int | None = 1,
         **ufunc_kwargs,
     ) -> FeatureArrayType | tuple[FeatureArrayType]:
-        """Apply a universal function to all features of the array."""
+        """
+        Apply a universal function to all features of the array.
+
+        Parameters
+        ----------
+        func : callable
+            The universal function to apply across features. The function should accept
+            an array of shape (samples, features) and return one or more arrays of shape
+            (samples, size), where sizes is defined by output_sizes.
+        output_dims : list[list[str]]
+            List of output core dimension names for each output.
+        output_dtypes : list[np.dtype], optional
+            List of output data types for each output. Required for Dask-backed arrays.
+        output_sizes : dict[str, int], optional
+            Mapping from output dimension names to their sizes.
+        output_coords : dict[str, list[str] | list[int]], optional
+            Mapping from output dimension names to their coordinates. If not provided,
+            defaults to sequential integer coordinates for each output dimension.
+        skip_nodata : bool, default=True
+            If True, NoData and NaN values will be skipped during prediction. This
+            speeds up processing of partially masked features, but may be incompatible
+            if estimators expect a consistent number of input samples.
+        nodata_output : float or int or tuple, optional
+            NoData samples in the input features will be replaced with this value in the
+            output features. If the value does not fit the array dtype returned by the
+            estimator, an error will be raised unless `allow_cast` is True. Defaults to
+            np.nan.
```

Comment:
It's not critical here, because the behavior of `nodata_output` is documented elsewhere pretty well, but it might be worth another sentence that explains why a tuple would be passed (e.g. `kneighbors`).  Something like "elements in a passed tuple correspond to each output array".  Probably this would be good to have if this functionality gets raised into the API.